### ISSUE TITLE
Remove the 'tabbed box' feature?

### ISSUE THREAD

**Issue by andrewbattista** [andrewbattista/lgwg#56]
The tabbed boxes seem to be an accessibility nightmare. We should test how these function with assistive technologies and then make a recommendation about whether or not they should be a part of our design principles moving forward. I don't think we can suppress or remove this feature from the LibGuides system though
